quickwit-directories/src/lib.rs

```diff
@@ -27,6 +27,7 @@ This crate contains all of the building pieces that make quickwit's IO possible.
 - The `HotDirectory` wraps another directory with a static cache.
 - The `CachingDirectory` wraps a Directory with a dynamic cache.
 - The `DebugDirectory` acts as a proxy to another directory to instrument it and record all of its IO.
+- The `BundleStorage` bundles together multiple files into a single file.
```

Comment:
Why in Directory? It's a storage.

Reply:
I moved it to to storage, but forgot about that comment